app/models/reason/AdviceContactPreference.scala

```diff
@@ -37,7 +37,7 @@ object AdviceContactPreference extends Enumerable.Implicits {
       RadioItem(
         content = Text(messages(s"adviceGiven.contact.${value.toString}")),
         value   = Some(value.toString),
-        id      = Some(s"contact_$index")
+        id      = Some(s"contact")
```

Comment:
So the original code here is correct, we want the ID for each row to be "contact_$index". The issue is that the error message needs to be changed to point to "contact_0"

Reply:
resolved!